tools/gcs-bucket-mover/gcs_bucket_mover/bucket_mover_service.py

```diff
@@ -0,0 +1,657 @@
+
+    policy.bindings.add(
+        role='roles/pubsub.publisher',
+        members=['serviceAccount:' + config.target_service_account_email])
+
+    policy = client.set_iam_policy(topic_path, policy)  # pylint: disable=no-member
+
+    spinner.write('{} {} added as a Publisher to topic: {}'.format(
+        CHECKMARK, config.target_service_account_email, topic_name))
+
+
+def _run_and_wait_for_sts_job(sts_client, target_project, source_bucket_name,
+                              sink_bucket_name):
+    """Kick off the STS job and wait for it to complete. Retry if it fails.
+
+    Args:
+        sts_client: The STS client object to be used
+        target_project: The name of the target project where the STS job will be created
+        source_bucket_name: The name of the bucket where the STS job will transfer from
+        sink_bucket_name: The name of the bucket where the STS job will transfer to
+    """
+
+    counter = 0
+    while counter <= 10:
+        print 'Moving from bucket {} to {}'.format(source_bucket_name,
+                                                   sink_bucket_name)
+        with yaspin(text='Creating STS job') as spinner:
+            sts_job_name = _execute_sts_job(sts_client, target_project,
+                                            source_bucket_name,
+                                            sink_bucket_name)
+            spinner.ok(CHECKMARK)
+
+        #Check every 10 seconds until STS job is complete
+        with yaspin(text='Checking STS job status') as spinner:
+            while True:
+                job_status = _check_sts_job(spinner, sts_client, target_project,
+                                            sts_job_name)
+                if job_status != sts_job_status.StsJobStatus.in_progress:
+                    break
+                sleep(10)
+
+        if job_status == sts_job_status.StsJobStatus.success:
+            print ''
+            break
+
+        counter += 1
+        print(
+            'There was an unexpected failure with the STS job. You can view the details in the'
+            ' cloud console.')
+        if counter > 10:
+            raise RuntimeError('Unable to complete STS job after 10 attempts.')
+        print(
+            'Waiting 60s and then trying again. If you choose to cancel this script, the buckets'
+            ' will need to be manually cleaned up.')
+        sleep(60)
+
+
+def _execute_sts_job(sts_client, target_project, source_bucket_name,
+                     sink_bucket_name):
+    """Start the STS job.
+
+    Args:
+        sts_client: The STS client object to be used
+        target_project: The name of the target project where the STS job will be created
+        source_bucket_name: The name of the bucket where the STS job will transfer from
+        sink_bucket_name: The name of the bucket where the STS job will transfer to
+
+    Returns:
+        The name of the STS job as a string
+    """
+
+    now = datetime.date.today()
+    transfer_job = {
```

Comment:
I'd really like to see some consistency with line breaks here.. some lines have the key and value on separate lines and some aren't.

Style is to have `'key': value,` on one line,

Reply:
I've contradicted yapf and formatted it nicely.